docs/src/projects/configuration.md

```diff
@@ -18,30 +18,33 @@ exit_first = true
 
 ## `sncast`
 
-### Defining Profiles in `Scarb.toml`
+### Defining Profiles in `snfoundry.toml`
 
 To be able to work with the network, you need to supply `sncast` with a few parameters —
 namely the rpc node url and an account name that should be used to interact with it.
 This can be done
 by either supplying `sncast` with those parameters directly [see more detailed CLI description,](../appendix/sncast.md)
-or you can put them into `Scarb.toml` file:
+or you can put them into `snfoundry.toml` file:
 
 ```toml
 # ...
-[tool.sncast.myprofile]
+[sncast.myprofile]
 account = "user"
 accounts-file = "~/my_accounts.json"
 url = "http://127.0.0.1:5050/rpc"
 # ...
 ```
 
-With `Scarb.toml` configured this way, we can just pass `--profile myprofile` argument to make sure `sncast` uses parameters
+With `snfoundry.toml` configured this way, we can just pass `--profile myprofile` argument to make sure `sncast` uses parameters
 defined in the profile.
 
 > 📝 **Note**
 > `Scarb.toml` file has to be present in current or any of the parent directories.
 > Alternatively, you can also point to `Scarb.toml` path with `--path-to-scarb-toml <PATH>` flag.
 
+> 📝 **Note**
+> If there is a profile with the same name in Scarb.toml, scarb will use this profile. If not, scarb will default to using the dev profile.
```

Comment:
Can we highlight that this only affects the `declare` and `script` commands?

Reply:
done in https://github.com/foundry-rs/starknet-foundry/pull/1647/commits/239fee695100fdb98bd9d0dabd8d0a24ede91e3d , please check if the working etc is correct